src/StaticAnalysisResultsInterchangeFormat/GrammarFiles/SarifFormat.g4

```diff
@@ -36,44 +36,44 @@ runLog:
 	/**
         @name {RunInfo}
         @summary {
-		A runInfo object describes the invocation of the static analysis tool that produced the issues
+		A runInfo object describes the invocation of the static analysis tool that produced the results
 		specified in the containing runLog object (�6.4). 
         }
     */
 	runInfo
 
     /**
-        @name {Issues}
-        @summary {The set of issues contained in an SARIF log.}
+        @name {Results}
+        @summary {The set of results contained in an SARIF log.}
     */
-    issues;
+    results;
 
 /**
     @className {ToolInfo}
     @summary {
-    Information about the tool or tool pipeline that generated the issues in this log.
+    Information about the tool or tool pipeline that generated the results in this log.
     }
 */
 toolInfo :
     /**
-        @name {ToolName}
+        @name {Name}
         @summary {
-        The name of the tool or tool pipeline that generated the issues in this log, e.g., FxCop.
+        The name of the tool or tool pipeline that generated the results in this log, e.g., FxCop.
         }
     */
-    toolName
+    name
 
     /**
-        @name {FullVersion}
+        @name {FullName}
```

Comment:
Comment is wrong. Spec says that `toolInfo.fullName` is "string containing the name of the tool along with its version and any other useful identifying information, such as its locale. "